src/process.js

```diff
@@ -0,0 +1,43 @@
+// Copied from https://github.com/hovancik/stretchly/blob/master/app/process.js - I think we should
```

Comment:
This is just a placeholder at this point. Pending some architecture discussions we can decide what to do with it